### PR TITLE
Enforce sentence case in ops

### PR DESCRIPTION
**Pull request by WillTaylor22**

### CODE REVIEW

File: app/services/sentence_case_enforcer.rb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      string = capitalise_accronyms(string)
+    end
+    string
+  end
+
+  def is_capitalised?(string)
+    string == string.upcase
+  end
+
+  def capitalise_accronyms(string)
```
Comment (agiamas) [inline app/services/sentence_case_enforcer.rb]: acronyms ? :)

File: config/environment.rb [hunk 2]
```diff
@@ -3,3 +3,5 @@
 
 # Initialize the Rails application.
 Rails.application.initialize!
+
+ActiveRecord::SchemaDumper.ignore_tables = ["spatial_ref_sys"]
```
Comment (agiamas) [inline config/environment.rb]: 👍

File: spec/services/create_opportunity_spec.rb [hunk 3]
```diff
@@ -16,6 +17,7 @@
       service_provider = create(:service_provider, id: '6')
       editor = create(:editor, service_provider_id: service_provider.id)
 
+      debugger
```
Comment (agiamas) [inline spec/services/create_opportunity_spec.rb]: you may want to remove this

File: spec/services/sentence_case_enforcer_spec.rb [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+RSpec.describe SentenceCaseEnforcer, type: :service do
+
+  before do
+    Word.create(text: 'usa', uppercase: true)
+    Word.create(text: 'france', capitalize: true)
+  end
+
+  describe '#call' do
+    it "enforces sentence case in title, ignoring exceptions" do
```
Comment (agiamas) [inline spec/services/sentence_case_enforcer_spec.rb]: these 3 examples look like one, maybe you want to test different things like ? and two letter lower case abbreviations at the end of a sentence with/without caps...

File: app/services/sentence_case_enforcer.rb [hunk 5]
```diff
@@ -0,0 +1,52 @@
+class SentenceCaseEnforcer
+
+  #
+  # Convert words that are in ALL CAPS to sentence case in the title, teaser and description.
+  #
+  def initialize(op_params)
+    @op_params = op_params
+  end
+
+  def call
```
Comment (agiamas) [inline app/services/sentence_case_enforcer.rb]: do you want to run this check for post created opps as well? Or just volume opps?

File: app/services/sentence_case_enforcer.rb (hunk 1 above)
Comment (WillTaylor22) [reply app/services/sentence_case_enforcer.rb]: Thanks

File: spec/services/sentence_case_enforcer_spec.rb (hunk 4 above)
Comment (WillTaylor22) [reply spec/services/sentence_case_enforcer_spec.rb]: Done

File: spec/services/create_opportunity_spec.rb (hunk 3 above)
Comment (WillTaylor22) [reply spec/services/create_opportunity_spec.rb]: Done

File: app/services/sentence_case_enforcer.rb (hunk 5 above)
Comment (WillTaylor22) [reply app/services/sentence_case_enforcer.rb]: From the discussion in standup, will leave it to just Volume Opps

**Pull request merged**